### ISSUE TITLE
Unable to find new packager codes with API

### ISSUE THREAD

**Issue by A1CY0N** [openfoodfacts/openfoodfacts-server#4405]
**Describe the bug**
I'm using the python library to connect to the api and some existing packer codes don't appear when you get the [list of packer codes here](http://world.openfoodfacts.org/packager-codes.json)

I have opened openfoodfacts/openfoodfacts-python#66 but I think it's more of a server issue.

**To Reproduce**
1. Go to https://world.openfoodfacts.org/product/6842420174016/chocolat
2. The packaging code is EMB 07113Y
3. Go to https://world.openfoodfacts.org/packager-codes or http://world.openfoodfacts.org/packager-codes.json
4. EMB 07113Y packaging code doesn't exist

**Number of products impacted**
I have noticed this on several recently added products

**Comment by svensven** [comment]
on the web site, it's probably not in the first 10000 results, and just on another page. 
it shows up for https://world.openfoodfacts.org/packager-codes?filter=7113.
can't speak for the api/python library though.

**Comment by A1CY0N** [comment]
Indeed, it seems that the python library should only get the first 10000 results. The filter option is very interesting but I don't think it's implemented in the library.

**Comment by svensven** [comment]
I had look through the server code, and it looks like adding a json=anything parameter is an alternative to the .json extension.

https://github.com/openfoodfacts/openfoodfacts-server/blob/08a94fc4d9d4a0f47a258da2faca79b39cadf7be/lib/ProductOpener/Display.pm#L585-L588

And https://world.openfoodfacts.org/packager-codes?filter=emb&json=1 does indeed seem to do the trick.
